src/select/Select.ts

```diff
@@ -70,6 +72,15 @@ export default class Select extends SelectBase<SelectProperties> {
 	private _open = false;
 	private _baseId = uuid();
 
+	private _getOptionLabel(option: any) {
+		const { getOptionLabel } = this.properties;
+		if (!option) {
```

Comment:
Should this not be passed to the `getOptionLabel` anyway, the custom handler may wish to do something with empty labels.

Reply:
good point 👍